demos/sjtwo/Servo/source/main.cpp

```diff
@@ -0,0 +1,57 @@
+#include "L1_Peripheral/lpc40xx/pwm.hpp"
+#include "L2_HAL/actuators/servo/servo.hpp"
+#include "utility/log.hpp"
+#include "utility/time.hpp"
+
+int main()
+{
+  LOG_INFO("Servo application starting...");
+
+  // Creating PWM on pin 2.0
+  sjsu::lpc40xx::Pwm p2_0(sjsu::lpc40xx::Pwm::Channel::kPwm0);
+
+  // Create servo class and and give it the PWM p2_0.
+  sjsu::Servo servo2_0(p2_0);
```

Comment:
```suggestion
  sjsu::Servo servo(p2_0);
```
Simply calling it servo should be enough.